tasks/main.yml

```diff
@@ -1,5 +1,18 @@
 ---
+## Tasks for Opendkim Ansible role
+
+- fail:
+    msg: "Bailing out. This role requires 'dkim_admin_email' to have an email address as value"
+  when: dkim_admin_email is not defined
```

Comment:
Won't work for `admin_email`. Also it would be good to print something if fallback to `admin_email` is used.

Reply:
I think it works ok for `admin_email`. Defaults links variable values: 
```
dkim_admin_email: "{{ admin_email | default ( omit ) }}"
```
So if you define `admin_email`, `dkim_admin_email` will have the same value and will pass the test. If you only define `dkim_admin_email` It will also work. I changed the failing message, mentionning also the option to define `admin_email`.